compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -751,7 +751,8 @@ static void addLowerToLLVMPasses(OpPassManager &passManager) {
   passManager.addNestedPass<LLVM::LLVMFuncOp>(createAddFastMathFlagsPass());
 }
 
-void buildLLVMCPUCodegenPassPipeline(OpPassManager &passManager) {
+void buildLLVMCPUCodegenStrategyRefinementPassPipeline(
```

Comment:
Nit: `StrategyRefinement` ? I am not following the naming here.

Reply:
StrategyRefinement because it picks the strategy to use. I'm bad at names...
Maybe I'll just call it initialization because it also runs the common codegen passes.